protocol/protobuf/proto/state_representation/space/cartesian/cartesian_state.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package state_representation.proto;
+
+import "state_representation/space/spatial_state.proto";
+
+message Vector3d {
+  double x = 1;
```

Comment:
You don't have to say `required` here?

Reply:
required and optional are not part of the `proto3` syntax, but nice looking out